packages/core/src/components/Checkbox/Checkbox.tsx

```diff
@@ -51,7 +53,12 @@ const Checkbox = ({
           onChange(newIsChecked);
         }}
         disabled={isDisabled}
-        ref={checkboxRef}
+        ref={(node: any) => {
```

Comment:
Should we use `HTMLInputElement` instead of `any` here?

Reply:
Sure, thank you